tasks/sentinel.yml

```diff
@@ -0,0 +1,15 @@
+
+- name: Disable selinux
```

Comment:
This is a rather huge change to include—I'm not sure I'd want to force anyone using this role to all the sudden not be able to use SELinux.